meetings/2017-04-19.md

```diff
@@ -0,0 +1,130 @@
+# Node Foundation CTC Meeting 2017-04-19
+## Links
+
+* **Audio recording**: <https://www.youtube.com/watch?v=S69FsEpLacc>
```

Comment:
This is the previous week's meeting - this one is https://www.youtube.com/watch?v=DemuVw4XRHQ